src/documentation/pages/components/pagination/examples/_additional-sizes_lumen_centurylink.vue

```diff
@@ -0,0 +1,352 @@
+        <button class="chi-button -flat -light -lg" aria-label="Page 2" type="button">2</button>
+        <button class="chi-button -flat -light -lg -active" aria-label="Page 3" type="button">3</button>
+        <button class="chi-button -flat -light -lg" aria-label="Page 4" type="button">4</button>
+        <button class="chi-button -flat -light -lg" aria-label="Page 5" type="button">5</button>
+        <button class="chi-button -icon -flat -light -lg" aria-label="Next page" type="button">
+          <div class="chi-button__content">
+            <i class="chi-icon icon-chevron-right" aria-hidden="true"></i>
+          </div>
+        </button>
+      </div>
+    </div>
+    <div class="chi-pagination__end"></div>
+  </div>
+</nav>
+
+<!-- xl -->
+<nav class="chi-pagination -inverse" role="navigation" aria-label="Pagination">
+  <div class="chi-pagination__content">
+    <div class="chi-pagination__start"></div>
+    <div class="chi-pagination__center">
+      <div class="chi-pagination__button-group chi-button-group">
+        <button class="chi-button -icon -flat -light -xl" aria-label="Previous page" type="button">
+          <div class="chi-button__content">
+            <i class="chi-icon icon-chevron-left" aria-hidden="true"></i>
+          </div>
+        </button>
+        <button class="chi-button -flat -light -xl" aria-label="Page 1" type="button">1</button>
+        <button class="chi-button -flat -light -xl" aria-label="Page 2" type="button">2</button>
+        <button class="chi-button -flat -light -xl -active" aria-label="Page 3" type="button">3</button>
+        <button class="chi-button -flat -light -xl" aria-label="Page 4" type="button">4</button>
+        <button class="chi-button -flat -light -xl" aria-label="Page 5" type="button">5</button>
+        <button class="chi-button -icon -flat -light -xl" aria-label="Next page" type="button">
+          <div class="chi-button__content">
+            <i class="chi-icon icon-chevron-right" aria-hidden="true"></i>
+          </div>
+        </button>
+      </div>
+    </div>
+    <div class="chi-pagination__end"></div>
+  </div>
+</nav>`,
+        vue: `<!-- For light backgrounds -->
+<ChiPagination :pages="5" :currentPage="3" size="xs" />
+<ChiPagination :pages="5" :currentPage="3" size="sm" />
+<ChiPagination :pages="5" :currentPage="3" size="md" />
+<ChiPagination :pages="5" :currentPage="3" size="lg" />
+<ChiPagination :pages="5" :currentPage="3" size="xl" />
+
+<!-- For dark backgrounds -->
+<ChiPagination :pages="5" :currentPage="3" size="xs" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="sm" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="md" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="lg" :inverse="true" />
+<ChiPagination :pages="5" :currentPage="3" size="xl" :inverse="true" />`
+      }
+    };
+  }
+})
+export default class AdditionalSizesLumenCenturylink extends Vue {}
```

Comment:
I believe L in link is uppercase, i.e. CenturyLink. Please change this in all examples